docker/pulsar/pom.xml

```diff
@@ -72,17 +71,27 @@
                 <configuration>
                   <images>
                     <image>
-                      <name>pulsar</name>
+                      <name>${docker.organization}/pulsar</name>
```

Comment:
This may affect the release of the image.

https://github.com/apache/pulsar/blob/master/docker/publish.sh